drivers/storage/portworx/components_test.go

```diff
@@ -11455,6 +11455,12 @@ func TestTelemetryEnableAndDisable(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, deployment.OwnerReferences, 1)
 	require.Equal(t, cluster.Name, deployment.OwnerReferences[0].Name)
+	memReq := deployment.Spec.Template.Spec.Containers[0].Resources.Requests[v1.ResourceMemory]
```

Comment:
Why don't we compare the deployment against a testspec like we do in other components?

Reply:
sounds good, resolved in another commit.